codegen/src/main/java/com/paypal/selion/plugins/CodeGeneratorMojo.java

```diff
@@ -111,8 +114,11 @@ public void setIosCustomElements(String[] elements) {
     public void setAndroidCustomElements(String[] elements) {
         androidCustomElements = Arrays.asList(elements);
     }
-    
-    
+
```

Comment:
you're okay. you fixed them all but this one. this reports show it -- https://www.codacy.com/app/SeLion/paypal-SeLion/pullRequest?prid=212622

Please don't fix this one. you follow the pattern the file had here. and we will make an exception